code/modules/reagents/reagent_containers.dm

```diff
@@ -484,6 +484,9 @@ var/list/LOGGED_SPLASH_REAGENTS = list(FUEL, THERMITE)
 
 	var/diff = air.temperature - reagents.chem_temp
 
+	if (!isturf(loc) && (air.pressure < 100))//low pressure environments slow down entropy, unless the item is laid directly onto the floor so space meat remains frozen until brought in
+		diff *= air.pressure
```

Comment:
`diff *= air.pressure/100` to get a multiplier or is this intended?

Reply:
oh thank god you're here